src/utils/sourcemaps.rs

```diff
@@ -176,10 +177,21 @@ impl SourceMapProcessor {
     pub fn new() -> SourceMapProcessor {
         SourceMapProcessor {
             pending_sources: HashSet::new(),
+            process_nonminified: false,
             sources: HashMap::new(),
         }
     }
 
+    /// Set whether we should add sourcemap references to non-minified sources.
+    /// This is useful for compile-to-JS languages/frameworks, that does not perform minification,
+    /// but still produce sourcemaps.
+    ///
+    /// Defaults to `false`.
+    pub fn process_nonminified(&mut self, allow: bool) -> &mut Self {
```

Comment:
Why return `&mut Self`?

Reply:
Builders convention that we use in most places in the CLI. Allows you to chain creating of complex structs. eg.

```rust
DifUpload::new(org.clone(), project.clone())
    .wait(matches.is_present("wait"))
    .search_paths(matches.values_of("paths").unwrap_or_default())
    .allow_zips(!matches.is_present("no_zips"))
    .filter_ids(ids);
```

Reply:
Makes sense :D